### ISSUE TITLE
A11y_Azure Data Studio_Query Plan (Preview)_ColorContrast: Color contrast ratio of keyboard focus of each node and its background is less than 3:1.

### ISSUE THREAD

**Issue by poojapatil36** [microsoft/azuredatastudio#20748]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### **GitHubTags:**

#A11yMAS; #A11yTCS; #A11yWCAG2.1; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #ColorContrast; #WCAG1.4.11; #FTP; #Win11

### **Environment Details:**
Application Name: Azure Data Studio
Windows Version: Windows 11

### **Repro Steps:**

1. Launch the "Azure data studio" application.
2. Connect with the provided SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. Observe that color contrast ratio of keyboard focus of each node and its background is 1.372:1 which is less than 3:1.

### **Actual:**

Color contrast ratio of keyboard focus of each node and its background is 1.372:1 which is less than 3:1.

### **Expected Result:**
Color contrast ratio of keyboard focus of each node and its background should be equal to or greater than 3:1.

### **Attachment:**
![A11y_Azure Data Studio_Query Plan (Preview)_ColorContrast](https://user-images.githubusercontent.com/97668775/194258475-ad9673cd-3e9a-4c6f-8314-8e7a03e04d25.png)

**Comment by poojapatil36** [comment]
Verified the issue in latest insider, issue is fixed.
Please find below screenshot for reference.
![20748_Fixed](https://user-images.githubusercontent.com/97668775/197967787-f864e5fb-0423-4dc8-9646-05d15dcf8600.png)

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
